{"name": "test_not_found_digit[23]", "status": "passed", "description": "\n            :test: тестовый id для подстановки в url\n        ", "parameters": [{"name": "test", "value": "23"}], "start": 1533894026294, "stop": 1533894027011, "uuid": "2a5fc646-becc-4b4e-82dc-7fb86d7fbaba", "historyId": "3311a8826002a21a3443b17e32a9fe47", "fullName": "tests.get.test_users.TestSingleUser#test_not_found_digit[23]", "labels": [{"name": "severity", "value": "minor"}, {"name": "host", "value": "User-pc"}, {"name": "thread", "value": "10832-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.get.test_users"}]}